<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1177</width>
    <height>703</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>主窗口</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>411</width>
      <height>491</height>
     </rect>
    </property>
    <property name="title">
     <string>结果:</string>
    </property>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>401</width>
       <height>401</height>
      </rect>
     </property>
     <row>
      <property name="text">
       <string>触头开距</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>超程</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>刚合速度</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>合闸速度</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>合闸时间</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>合闸顺序</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>弹跳时间</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>刚分速度</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>分闸速度</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>分闸时间</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>不同期性</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>分闸顺序</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>A</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>B</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>C</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QPushButton" name="btn_begin_mesurement">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>510</y>
      <width>411</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>开始测试</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1177</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件（&amp;F）</string>
    </property>
    <addaction name="separator"/>
    <addaction name="action_newFile"/>
    <addaction name="action_open"/>
    <addaction name="separator"/>
    <addaction name="action_save"/>
    <addaction name="action_saveAs"/>
    <addaction name="separator"/>
    <addaction name="action_print"/>
    <addaction name="separator"/>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>通讯设置(&amp;C)</string>
    </property>
    <addaction name="action_serialportSet"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="sensor_check">
    <property name="title">
     <string>传感器校验</string>
    </property>
    <addaction name="action_sensor_check"/>
   </widget>
   <widget class="QMenu" name="test_object">
    <property name="title">
     <string>试验对象</string>
    </property>
    <addaction name="action_test_object"/>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="action_about"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>机械寿命试验</string>
    </property>
    <addaction name="action_life_test"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="sensor_check"/>
   <addaction name="test_object"/>
   <addaction name="menu_3"/>
   <addaction name="menu_5"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="allowedAreas">
    <set>Qt::TopToolBarArea</set>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_newFile"/>
   <addaction name="action_open"/>
   <addaction name="action_save"/>
   <addaction name="separator"/>
   <addaction name="action_serialportSet"/>
   <addaction name="action_test_object"/>
   <addaction name="action_life_test"/>
   <addaction name="separator"/>
   <addaction name="action_print"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_open">
   <property name="icon">
    <iconset resource="background.qrc">
     <normaloff>:/new/prefix_icon/icon/Open Folder.png</normaloff>:/new/prefix_icon/icon/Open Folder.png</iconset>
   </property>
   <property name="text">
    <string>打开(&amp;O)</string>
   </property>
  </action>
  <action name="action_save">
   <property name="icon">
    <iconset resource="background.qrc">
     <normaloff>:/new/prefix_icon/icon/save.png</normaloff>:/new/prefix_icon/icon/save.png</iconset>
   </property>
   <property name="text">
    <string>保存（&amp;S）</string>
   </property>
  </action>
  <action name="action_saveAs">
   <property name="icon">
    <iconset resource="background.qrc">
     <normaloff>:/new/prefix_icon/icon/save-as.png</normaloff>:/new/prefix_icon/icon/save-as.png</iconset>
   </property>
   <property name="text">
    <string>另存为(&amp;A)</string>
   </property>
  </action>
  <action name="action_serialportSet">
   <property name="icon">
    <iconset resource="background.qrc">
     <normaloff>:/new/prefix_icon/icon/serial-port .png</normaloff>:/new/prefix_icon/icon/serial-port .png</iconset>
   </property>
   <property name="text">
    <string>串口设置(&amp;S)</string>
   </property>
  </action>
  <action name="action_about">
   <property name="icon">
    <iconset resource="background.qrc">
     <normaloff>:/new/prefix_icon/icon/about.png</normaloff>:/new/prefix_icon/icon/about.png</iconset>
   </property>
   <property name="text">
    <string>关于(&amp;A)</string>
   </property>
  </action>
  <action name="action_print">
   <property name="icon">
    <iconset resource="background.qrc">
     <normaloff>:/new/prefix_icon/icon/print.png</normaloff>:/new/prefix_icon/icon/print.png</iconset>
   </property>
   <property name="text">
    <string>打印(&amp;P)</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="icon">
    <iconset resource="background.qrc">
     <normaloff>:/new/prefix_icon/icon/exit.png</normaloff>:/new/prefix_icon/icon/exit.png</iconset>
   </property>
   <property name="text">
    <string>退出(&amp;E)</string>
   </property>
  </action>
  <action name="action_newFile">
   <property name="icon">
    <iconset resource="background.qrc">
     <normaloff>:/new/prefix_icon/icon/new.png</normaloff>:/new/prefix_icon/icon/new.png</iconset>
   </property>
   <property name="text">
    <string>新建（&amp;N）</string>
   </property>
  </action>
  <action name="action_sensor_check">
   <property name="icon">
    <iconset resource="background.qrc">
     <normaloff>:/new/prefix_icon/icon/sensor.png</normaloff>:/new/prefix_icon/icon/sensor.png</iconset>
   </property>
   <property name="text">
    <string>传感器系数校验</string>
   </property>
  </action>
  <action name="action_test_object">
   <property name="icon">
    <iconset resource="background.qrc">
     <normaloff>:/new/prefix_icon/icon/list_switch.png</normaloff>:/new/prefix_icon/icon/list_switch.png</iconset>
   </property>
   <property name="text">
    <string>试验对象参数设置</string>
   </property>
  </action>
  <action name="action_life_test">
   <property name="icon">
    <iconset resource="background.qrc">
     <normaloff>:/new/prefix_icon/icon/Lifetime.png</normaloff>:/new/prefix_icon/icon/Lifetime.png</iconset>
   </property>
   <property name="text">
    <string>机械寿命试验参数设置</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="background.qrc"/>
 </resources>
 <connections/>
</ui>
